src/components/chat/chat-panel.tsx

```diff
@@ -7,13 +7,18 @@ import { CommentCard } from "./comment-card";
 import "./chat-panel.scss";
 import { UserModelType } from "../../models/stores/user";
 
+export interface ICommentData {
+  comment: string;
+  timePosted: string | number;
+  user: UserModelType;
+}
```

Comment:
Put `ICommentData` interface in `comment-card.ts` to avoid circular imports (`ChatPanel` <-> `CommentCard`) -- `ChatPanel` already imports `CommentCard`. In general, shared types should go into the most primitive module.